src/Servers/Kestrel/test/FunctionalTests/Http2/HandshakeTests.cs

```diff
@@ -81,7 +81,7 @@ public void TlsAndHttp2NotSupportedOnWin7()
         [ConditionalFact]
         [OSSkipCondition(OperatingSystems.MacOSX, SkipReason = "Missing SslStream ALPN support: https://github.com/dotnet/corefx/issues/30492")]
         [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/10428", Queues = "Debian.8.Amd64;Debian.8.Amd64.Open")] // Debian 8 uses OpenSSL 1.0.1 which does not support HTTP/2
-        [MinimumOSVersion(OperatingSystems.Windows, WindowsVersions.Win81)]
+        [MinimumOSVersion(OperatingSystems.Windows, WindowsVersions.Win10)]
         public async Task TlsAlpnHandshakeSelectsHttp2From1and2()
```

Comment:
Yeah, it's strange you didn't have to change the other test (TlsAlpnHandshakeSelectsHttp2).

Reply:
It's not https? So it doesn't use the logic in HttpsConnectionMiddleware?

Reply:
It is https. The only difference is that it starts with Http2 instead of Http1AndHttp2.

Reply:
Ah oops, I should have read the test before commenting. Originally in the linked issue, we decided that when setting the protocol to Http2 is done explicitly, we shouldn't trigger the new downgrade logic. Since the test specifically does that, it didn't need to be updated. 

However, this is going to depend on our conversation https://github.com/dotnet/aspnetcore/pull/22859/files#r440356130. If we decide to change the logic to throw or downgrade on 8.1 when Http2 is explicitly selected, then we'll need to update this test.